frontend/src/metabase/query_builder/components/filters/modals/BulkFilterModal/BulkFilterModal.tsx

```diff
@@ -63,20 +63,20 @@ const BulkFilterModal = ({
   );
 
   const handleAddFilter = useCallback((filter: Filter) => {
-    setQuery(filter.add());
+    setQuery(filter.add().rootQuery());
```

Comment:
I see that we call `rootQuery` in some places in our app so this nested thing was kind of expected by the existing code, but we didn't know that when we were implementing the bulk filter modal.